js/app.js

```diff
@@ -122,52 +63,57 @@ window.addEventListener('DOMContentLoaded', function() {
     showElement(messageArea, false);
   }
 
+  function displayResult(result) {
+    console.log('Sync successfully finished!', result);
+    var nbDeleted = 0;
+    var nbUpdated = 0;
+    var nbAdded = 0;
+    for (var op of result) {
+      switch (op.action) {
+        case 'created':
+          nbAdded++;
+          break;
+        case 'updated':
+          nbUpdated++;
+          break;
+        case 'deleted':
+          nbDeleted++;
+          break;
+      }
+    }
+    var message =
+    showMessage(
+      '<h2>Sync successfully finished:</h2>'+
+      'Summary:<br>' +
+      `<ul>
+        <li>${nbAdded} contact(s) added</li>
+        <li>${nbUpdated} contact(s) updated</li>
+        <li>${nbDeleted} contact(s) deleted</li>
+      </ul>`
+    );
```

Comment:
why don’t you put everything in a single string literal?

```javascript
    var message = showMessage(`
      <h2>Sync successfully finished:</h2>
      Summary:<br>
      <ul>
        <li>${nbAdded} contact(s) added</li>
        <li>${nbUpdated} contact(s) updated</li>
        <li>${nbDeleted} contact(s) deleted</li>
      </ul>
    `);
```
However, if that’s more than a debug message, we’ll need a proper l10n support — which will handle the pluralization properly.